components/brave_wallet/browser/BUILD.gn

```diff
@@ -309,6 +312,16 @@ source_set("transaction") {
   ]
 }
 
+generated_types("generated_json_rpc_responses") {
+  sources = [ "json_rpc_responses.idl" ]
+  root_namespace = "json_rpc_responses::%(namespace)s"
```

Comment:
It's better to put all these generated types under brave_wallet like
`brave_wallet::%(namespace)s`, otherwise it will very likely to be conflicted.